ignite/contrib/metrics/precision_recall_curve.py

```diff
@@ -21,7 +21,6 @@ class PrecisionRecallCurve(EpochMetric):
     by accumulating predictions and the ground-truth during an epoch and applying
     `sklearn.metrics.precision_recall_curve <https://scikit-learn.org/stable/modules/generated/
     sklearn.metrics.precision_recall_curve.html#sklearn.metrics.precision_recall_curve>`_ .
-
```

Comment:
Please revert these changes in the docstring.

Reply:
Yes, I will no problem.